Mods/ArchipelagoPlayer.SC2Mod/enUS.SC2Data/LocalizedData/GameStrings.txt

```diff
@@ -1838,6 +1851,9 @@ Button/Tooltip/AP_CorrosiveAcidDevourer=Attacks cause the target and nearby enem
 Button/Tooltip/AP_Corruption=Covers the target enemy unit in Zerg slime, increasing the damage taken by 20% for <d time="30"/>.
 Button/Tooltip/AP_Corruptor=Anti-air flyer, specializing in anti-massive. Can morph into advanced fliers.<n/><n/><c val="ffff8a">Can attack air units.</c>
 Button/Tooltip/AP_CorruptorPassive=This building enables you to morph Corruptors.
+Button/Tooltip/AP_CorruptorMonstrousResilience=Increases Corruptor life by 100.
+Button/Tooltip/AP_CorruptorConstructRegeneration=Increases Corruptor health regeneration by 3 per second.
+Button/Tooltip/AP_CorruptorIncubateScourge=Corruptors spawn 2 Scourge when killed (3 with Swarm Scourge).

```

Comment:
Technically you could change the tooltip by the Swarm Scourge upgrade by setting the button tooltip by upgrade effect (but it doesn't really matter)

Reply:
I was considering that, but ultimately I decided it would make more sense to clearly state that there's an interaction with another upgrade; that way, no one is confused about why they're suddenly getting an extra Scourge, or even something like "last run I was getting 3 scourge and now it's only 2, what's going on?"

I'm not committed to the choice though, I can change it if you prefer it the other way.